Settings.js

```diff
@@ -122,6 +128,7 @@ function init_settings(){
 	for(let i = 0; i < token_settings.length; i++) {
 		let setting = token_settings[i];
 		let currentValue = window.TOKEN_SETTINGS[setting.name];
+		console.log('CURRENT VALUE: ', currentValue);
```

Comment:
@vlaminck  I will remove the console.log as well, but what do you think of setting the settings to ```false``` here instead?  At least for the time being, so they all have a value from the get go.